genie-core/src/main/java/com/netflix/genie/core/services/impl/LocalJobKillServiceImpl.java

```diff
@@ -57,6 +61,8 @@
     private final Executor executor;
     private final boolean runAsUser;
     private final ApplicationEventPublisher eventPublisher;
+    private final File baseWorkingDir;
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I believe there is already an `ObjectMapper` available as a bean within the spring context you should be able to inject here rather than creating another one. Also makes it hard to test with this buried inside.